### PR TITLE
Replace DAO info with subgraph calls

### PR DESCRIPTION
**Pull request by neokry**
## Description

- replaces zora api calls with subgraph calls for dao info

## Motivation & context

- reduce reliance on zora apis
- prepare for L2 deployments
- prepare for new protocol changes

## Code review

- review this code against the `subgraph-token-info` branch if that code has not been merged yet
- does dao info match with zora api data?
- does dao data like ownership count and my DAOs queries properly display
- note: ownership count might be off by one due to zora api counting the 0x0 address as an owner when the subgraph does not

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have done a self-review of my own code
- [x] Any new and existing tests pass locally with my changes
- [x] My changes generate no new warnings (lint warnings, console warnings, etc)

### CODE REVIEW

**Comment by isaaczaak** [comment]
Screenshot 2023-06-16 at 4 57 21 pm

noticing that the `Owners` count is off by 2 in the subgraph build - besides the 0x00 burn address, do you know which other address is the subgraph not counting?

@neokry

**Comment by isaaczaak** [comment]
Screenshot 2023-06-16 at 4 59 50 pm

left: subgraph
right: zora api

a slight discrepancy in the order of the DAOs shown in the dropdown, looks like the subgraph one shows DAOs according to most recently joined/created?

**Comment by neokry** [comment]
> Screenshot 2023-06-16 at 4 57 21 pm
> noticing that the `Owners` count is off by 2 in the subgraph build - besides the 0x00 burn address, do you know which other address is the subgraph not counting?
> 
> @neokry

fixed now. subgraph changes were not merged in from token branch

**Comment by neokry** [comment]
> Screenshot 2023-06-16 at 4 59 50 pm
> left: subgraph right: zora api
> 
> a slight discrepancy in the order of the DAOs shown in the dropdown, looks like the subgraph one shows DAOs according to most recently joined/created?

the zora api call didn't have any ordering so there was an implicit order happening on the backend that I'm not sure how to replicate or if we should replicate it. if you have a suggestion for an explicit ordering I can add it in. The implicit ordering from the subgraph is date joined I think (like you mentioned).

File: apps/web/src/modules/create-proposal/components/UpgradeCard.tsx [hunk 1]
```diff
@@ -82,7 +61,7 @@ export const UpgradeCard = ({
             borderRadius={'small'}
             className={image}
           >
-            {!!imgurl && !error && (
+            {!!imgurl && (
```
Comment (nadiakiamilev) [inline apps/web/src/modules/create-proposal/components/UpgradeCard.tsx:64]: nit: do we need this conditional anymore if the imgurl is a hardcoded value now?

**Review by nadiakiamilev — APPROVED** [review]

File: apps/web/src/data/subgraph/requests/tokenQuery.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import omitBy from 'lodash/omitBy'
+
+import { sdk } from 'src/data/subgraph/client'
+
+export interface Token {
+  id: string
+  name?: string
+  image?: string
+  description?: string
+  owner?: string
```
[the remark this answers is not in the document]
Comment (neokry) [reply apps/web/src/data/subgraph/requests/tokenQuery.ts]: will clean up these types in a followup @nadiakiamilev. wanted to keep using the same request types as the zora api calls to make the switch easier for now

**Pull request merged**